app/processing/siti-agri/schemas/invoice-line.js

```diff
@@ -8,5 +8,6 @@ module.exports = Joi.object({
   description: Joi.string().regex(/^[A-Z]{1}\d{2}\s-\s.+$/).required(),
   value: Joi.number().required(),
   convergence: Joi.boolean().optional(),
-  deliveryBody: Joi.string().regex(/^[A-Z]{2}\d{2}$/).required()
+  deliveryBody: Joi.string().regex(/^[A-Z]{2}\d{2}$/).required(),
+  marketingYear: Joi.number().integer().greater(2015).less(2099).required()
```

Comment:
Do you need to add the .less() part here?

Reply:
Just added it so it's consistent with what we're already validating on the header level.

It's an interesting point.  I feel we need something, so let's say we get a crazy value like `33223` we probably want to reject that and not just accept it.  But we don't want to lock it down so much that we have to keep changing the schema.

I guess less than 2099 was agreed on for a reason and I didn't have a strong case for an alternative

Reply:
For consistency sake I'd say leave it as is. The less than value doesn't matter so much, so long as it's not too constrictive like you said